diff -burN elisa-plugins-bad-0.5.17.orig/elisa/plugins/poblesec/player_video.py elisa-plugins-bad-0.5.17/elisa/plugins/poblesec/player_video.py
--- elisa-plugins-bad-0.5.17.orig/elisa/plugins/poblesec/player_video.py	2008-11-03 09:34:09.000000000 -0600
+++ elisa-plugins-bad-0.5.17/elisa/plugins/poblesec/player_video.py	2008-11-05 11:52:20.000000000 -0600
@@ -441,7 +441,9 @@
                 if os.path.exists(sub_file):
                     sub_uri = MediaUri("file://%s" % sub_file)
                     self.pipeline.set_property('suburi', sub_uri)
-                    self.pipeline.set_property('subtitle-font-desc', "Liberation Sans")
+                    font = self.config['subtitles_font']
+                    font_size = self.config['subtitles_font_size']
+                    self.pipeline.set_property('subtitle-font-desc', "%s %s" % (font, font_size))
                     self.info("Loaded subtitles at %r", sub_uri)
                     found = True
                     break
@@ -672,6 +674,8 @@
     default_config = {'visualization': visualization,
                       'osd_timeout': '3',
                       'audio_sink' : '',
+                      'subtitles_font' : 'Liberation Sans',
+                      'subtitles_font_size' : '14'
                       }
     return default_config
 
@@ -685,7 +689,9 @@
                                  'auto-hide. Set to 0 to deactivate that feature',
                   'audio_sink' : 'give the name of the gstreamer audio sink' \
                                  'elisa should use. If not set autoaudiosink'\
-                                 'will be used.'
+                                 'will be used.',
+                  'subtitles_font' : 'subtitles font',          
+                  'subtitles_font_size' : 'subtitles font size'
                   }
     default_config = player_default_config()
     
@@ -720,7 +726,7 @@
         self.player_osd.connect('hidden', self._stop_monitoring)
 
 
-        self.player = self.PlayerClass(None)
+        self.player = self.PlayerClass(self.default_config)
         self.player.set_drawable(self.background)
         self.player.connect('status-changed',self._player_status_cb)
         
